libs/server/auth/core/application-services/src/lib/auth.facade.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { QueryBus } from '@nestjs/cqrs';
+import { AuthResponse } from '@pimp-my-pr/shared/domain';
+import { GetGithubAccessTokenQuery } from './queries/get-github-access-token/get-github-access-token.query';
+
+@Injectable()
+export class AuthFacade {
+  constructor(private queryBus: QueryBus) {}
+
+  getGithubAccessToken(githubCode: string): Promise<AuthResponse> {
```

Comment:
In this layer name "Response" shouldn't appear as it could be run from CLI or something else